ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/ShopperInsightsClientV2.kt

```diff
@@ -123,4 +125,26 @@ class ShopperInsightsClientV2 internal constructor(
     fun isVenmoAppInstalled(context: Context): Boolean {
         return deviceInspector.isVenmoInstalled(context)
     }
+    /**
+     * Updates the customer session with the given session ID.
+     */
+    fun updateCustomerSession(
```

Comment:
What are your thoughts around moving this new function under `createCustomerSession()`. It might help with readability to group the create, update, get recommendations functions together.